frontend/public/components/resource-quota.jsx

```diff
@@ -57,17 +70,36 @@ export const getQuotaResourceTypes = (quota) => {
   return _.keys(specHard).sort();
 };
 
-const getResourceUsage = (quota, resourceType) => {
+const getResourceUsage = (quota, resourceType, isACRQ = false) => {
   const isCluster = isClusterQuota(quota);
   const statusPath = isCluster ? ['status', 'total', 'hard'] : ['status', 'hard'];
   const specPath = isCluster ? ['spec', 'quota', 'hard'] : ['spec', 'hard'];
   const usedPath = isCluster ? ['status', 'total', 'used'] : ['status', 'used'];
+  let totalUsedPath;
+  let totalUsed;
+  let used;
+
+  // ACRQ is a cluster-scoped resource that behaves like a namespaced resource
+  // It has cluster-scoped data as well as namespaced data
+  // "used" is the data for the current namespace and "totalUsed" is the cluster-scoped data
+  if (isACRQ) {
+    used = _.get(quota, ['status', 'namespaces']);
+    const currentNamespace = sessionStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY);
```

Comment:
This isn't how you want to get the current namespace. It should be part of the URL.

We also need to make sure we're handling "All Projects."